meson.build

```diff
@@ -7,7 +7,7 @@ project(
     meson_version: '>=1.0.1',
     default_options: [
         'buildtype=release',
-        'c_std=c99'
+        'c_std=c11'
```

Comment:
This was only required to add `_Static_assert`, which isn't required but definitely nice to have.

Since we don't distribute any type of C-API I'm not sure we really need to support all the way back to C99. MSVC has been better in recent history with supporting newer standards like C11, so I think we should start taking advantage of that

https://devblogs.microsoft.com/cppblog/c11-and-c17-standard-support-arriving-in-msvc/